CodingGuidelines.md

```diff
@@ -0,0 +1,26 @@
+# Coding Guidelines
+
+## C# General
+
+In order to improve team wide readablity and lower onboarding time we agreed to stick to defined coding guidelines. These coding guidelines help us writing better and more coherent code, both on an individual level, as well as across individual developers since the style becomes more aligned.
+
+The coding guidelines can be found [here](https://csharpcodingguidelines.com/).
+Feel free to download the .PDF for later references.
+
+## IDE
+
+At Triple we are not tied to a single operating system, and so it happens that also among the .NET team there is cross-platform development. In addition to the free choice of IDE, there are quite a few flavors possible. However, in order to reduce startup and onboarding time, we should strive to facilitate cross-platform development as best as possible. This includes the generation of a README file, with instructions for environment specific setups. At the least, it should contain the instructions for the cross-platform editor  [Visual Studio Code](https://code.visualstudio.com/).
```

Comment:
Well, we try not to be tied to an OS, but in reality we are stuck with IIS for a lot (most? all?) of our projects, and that just doesn't run on linux as far as I know.  
I think we should consider recommending using windows as your development OS, but certainly not mandate it.

Reply:
So far I have worked on 3 different projects, all cross-platform ones. Seen the amount of new project we initiate, and the direction .NET is going into, we would do well anticipating this. 
True, older projects are still bound to Windows, and it would not be valuable to make these cross-platform, but for the newer ones we should keep this in mind - especially when onboarding external devs.

Reply:
However, I will implement the suggestion made in the meeting - if you work on a different OS and want to run it too, add to the readme how to initiate the project on your OS/IDE.